test/rpc-spec.js

```diff
@@ -1,82 +1,79 @@
-var assert = require('assert');
-var producer = require('../src/index')().producer;
-var consumer = require('../src/index')().consumer;
-var uuid = require('node-uuid');
-var docker = require('./docker');
-var utils = require('../src/modules/utils');
+const assert = require('assert');
+const producer = require('../src/index')().producer;
+const consumer = require('../src/index')().consumer;
+const uuid = require('node-uuid');
+const docker = require('./docker');
+const utils = require('../src/modules/utils');
 
-var fixtures = {
+const fixtures = {
   queues: ['rpc-queue-0', 'rpc-queue-1', 'rpc-queue-2', 'rpc-queue-3']
 };
 
-describe('Producer/Consumer RPC messaging:', function() {
-  before(function() {
-    this.timeout(20000);
-    return docker.start();
-  });
+/* eslint func-names: "off" */
+/* eslint prefer-arrow-callback: "off" */
+describe('Producer/Consumer RPC messaging:', function () {
+  this.timeout(20000);
```

Comment:
just observation this would be good to be dictated from makefile and not in tests. for someone this 2000 might be to long and for someone too short.